com.github.thetumultuousunicornofdarkness.cpu-x.yaml

```diff
@@ -0,0 +1,67 @@
+app-id: com.github.thetumultuousunicornofdarkness.cpu-x
+runtime: org.gnome.Platform
+runtime-version: "44"
+sdk: org.gnome.Sdk
+command: cpu-x
+
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # Needs to save files locally
+  - --metadata=X-DConf=migrate-path=/com/github/thetumultuousunicornofdarkness/cpu-x/
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
this is not allowed by default.

Reply:
`flatpak-spawn --host` command is required to start `cpu-x-daemon`: this program needs a read access to CPU MSRs (`/dev/cpu`) and DMI table (`/dev/mem`). Privileges escalation is done by using `pkexec` on the host, thus this permission.

This need is similar to [MissionCenter](https://github.com/flathub/io.missioncenter.MissionCenter/blob/master/io.missioncenter.MissionCenter.json) that is present on Flathub by the way.

Reply:
https://docs.flathub.org/docs/for-app-authors/linter/

Reply:
Understood. PR opened: https://github.com/flathub/flatpak-builder-lint/pull/192